### PR TITLE
Bug 1337927 - Add version support to schema filenames

### PR DESCRIPTION
**Pull request by mreid-moz**
Change the repository layout to allow multiple versions of each document's schema. 

See also [related decoder changes](https://github.com/mozilla-services/lua_sandbox_extensions/pull/126) in the ingestion code.

I populated `core` schema version 1 using a slightly modified version of the initial commit (with fields ordered to match the current version for easier `diff`ing), and all others up to v8 as copies of the current schema (with version pinned to a specific value).

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla-services/mozilla-pipeline-schemas/44)

### CODE REVIEW

**Comment by mreid-moz** [comment]
Note that I did not add a version to the "vacuous" schema as that doesn't seem to make much sense. Do we even need that schema or should I just drop it altogether?

**Comment by trink** [comment]
rm vacuous.schema.json

**Comment by mreid-moz** [comment]
Done

File: schemas/telemetry/core/core.1.schema.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (trink) [inline schemas/telemetry/core/core.1.schema.json]: core.1.schema.json	failed to parse offset:745 Missing a name for object member.

Comment (mreid-moz) [inline schemas/telemetry/core/core.1.schema.json]: Fixed.

**Pull request merged**